vendor/github.com/opencontainers/runtime-spec/specs-go/config.go

```diff
@@ -692,4 +692,7 @@ type LinuxIntelRdt struct {
 	// The unit of memory bandwidth is specified in "percentages" by
 	// default, and in "MBps" if MBA Software Controller is enabled.
 	MemBwSchema string `json:"memBwSchema,omitempty"`
+
+	// Monitoring flag that's indicated to monitor llc-occupancy or total and local memory bandwidth.
```

Comment:
How about moving the dependent vendor code into a separate prerequisite patch? When this changes in https://github.com/opencontainers/runtime-spec/pull/1076 have been merged, the patch could be ignored.